AK/FixedArray.h

```diff
@@ -70,43 +93,37 @@ class FixedArray {
         VERIFY(index < m_size);
         return m_elements[index];
     }
-
```

Comment:
Removing the newline here is pointless

Reply:
As noted above, my reasoning is that I think since these functions really should be grouped together, the newline should be removed. But if you think that multi-line functions should under all circumstances have newlines between them, I will add the newline there.